db/cart_items.js

```diff
@@ -0,0 +1,108 @@
+const client = require("./client");
+
+
+async function addToCartItems({
```

Comment:
I think addItemToCart( ) describes the function better (and is also easier for me to say lol)